crates/aptos-dkg/src/sigma_protocol/traits.rs

```diff
@@ -238,6 +240,12 @@ where
         dst,
     );
 
+    // Append the public statement (the image of the witness) to the transcript
+    <merlin::Transcript as fiat_shamir::SigmaProtocol<E, H>>::append_sigma_protocol_public_statement(
```

Comment:
Wondering... suppose you implement a Schnorr proof using this framework.

Will this append $x G$ or will it append $(G, xG)$?

If it only appends $xG$, when would $G$ get appended?

If never, that would be insecure.

Reply:
Nope, nope, nope. This is not the right security mindset: we have to prioritize making the API trivial to use safely. Otherwise, we will shoot ourselves in the foot.

Let's fix this in the next PR, not sure what the best way of doing it is. You decide. But it has to be secure by default.